src/new-components/banner/banner.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react'
+import { Box } from '../box'
+import { Columns, Column } from '../columns'
+
+import styles from './banner.module.css'
+
+export type BannerTone = 'info' | 'promotion'
+
+type BannerProps = {
+    id?: string
+    /**
+     * The tone of the Banner. Affects the background color and the outline.
+     */
+    tone: BannerTone
+    /**
+     * The icon that should be added inside the Banner.
+     */
+    icon: React.ReactChild
+    /**
+     * The title to be displayed at the top of the Banner.
+     */
+    title: string
+    /**
+     * An optional description to be displayed inside the Banner.
+     */
+    description?: string
+    /**
+     * An optional action to displayed inside the Banner.
+     */
+    action?: React.ReactChild
+}
+
+function Banner({ id, tone, icon, title, description, action }: BannerProps) {
+    return (
+        <Box
+            id={id}
+            role="banner"
+            aria-live="polite"
+            borderRadius="standard"
+            className={[styles.banner, styles[`banner-${tone}`]]}
+        >
+            <Columns space="medium" alignY="center">
+                <Column width="content" aria-hidden>
+                    <Box display="flex" className={styles.icon} aria-hidden>
```

Comment:
I do not see this `.icon` class in the css module source. I think we can remove this. Also, why do we need this `display="flex"` for? Furthermore, do we really need the wrapper box around the icon? (not sure about that last one, just asking)

Reply:
Let me test all of this. I think I was playing with different paddings and the like and forgot to clean things up.